elife/jenkins-scripts/wait_packagist.sh

```diff
@@ -22,3 +22,7 @@ while : ; do
         break
     fi
 done
+
+cache_file_for_package_revisions=~/.cache/composer/repo/https---packagist.org/provider-"${package/\//\$}".json
```

Comment:
`/usr/lib/composer/.cache` (https://github.com/elifesciences/builder-base-formula/blob/master/elife/composer.sls#L5 - might want to move to pillar?)

Reply:
I'd like to keep this running as-is rather than making it a template for a script, so I'll try an environment variable